data/coalition/coalition missions.txt

```diff
@@ -143,3 +143,847 @@ mission "Coalition License Patch"
+			label end
+			`	Night falls, but there hasn't been any decrease to the activities here, as the hordes of captains and workers still populate the docking area and the spaceport.`
+			`	Around midnight, Chikee arrives at your ship, and you show him to his bunk. He is only bringing a single, old rugged bag with him.`
+				accept
+	on visit
+		dialog `You land on <planet>, but realize that your escort carrying Chikee hasn't entered the system yet. Better depart and wait for it.`
+	on complete
+		"coalition jobs" ++
+		event "chikee break" 62
+		dialog `You drop Chikee off on <planet>, and wish him good luck on the new job.`
+
+
+
+event "chikee break"
+
+
+
+mission "Kimek Student 2"
+	name "Transport Chikee"
+	description "Chikee, a young Kimek, is studying to become a Heliarch. Transport him to <destination>, where he was offered an internship and hopes to be awarded with a recommendation letter."
+	deadline
+	passengers 1
+	to offer
+		has "event: chikee break"
+	source "Second Rose"
+	destination "New Finding"
+	on offer
+		clear "event: chikee break"
+		conversation
+			`While strolling through the spaceport, you're flagged down by a familiar face. It's Chikee, the Kimek student whom you brought here months ago.`
+			`	He excitedly greets you and talks about how well his studies have been going. You notice that he now has a translation device of his own, positioned by his neck.`
+			`	"Purchased it recently, I did, so that hindered by language barriers, I am not," he says. "Excellent timing you have, Captain. Looking for transportation, once again I am."`
+			`	He tells you how his academic successes have earned him an internship in one of the laboratories on <destination>, and that he needs someone to transport him there by <date> so that he may begin working right away.`
+			`	"Better, the payment is, but also hoping for a recommendation letter, I am," he says, explaining that such letters from a Heliarch laboratory could help him significantly in joining their ranks.`
+			choice
+				`	"Sounds like you've worked pretty hard for this. Let's go to my ship, I'll take you there."`
+					goto accept
+				`	"You still haven't paid me for last time, though."`
+			`	He freezes up for a bit, not responding.`
+			`	"Just enough money to rent an apartment there, I have. Save up until the laboratory pays me, I must," he says. "Scamming you, I am not, Captain. Thankful I am, and pay you handsomely, I will."`
+			choice
+				`	"Alright, I'll wait again. Let's go to my ship."`
+					goto accept
+				`	"Sorry, but I'm not one to be fooled twice."`
+					decline
+			label accept
+			`	He thanks you, and this time follows you right away, as he already has his bags prepared.`
+			`	"Hoping to join the research teams on Ring of Wisdom, I am," he says as you two walk to your ship, explaining the reason he needs to study so much is because he isn't aiming for the more military-focused side of the government.`
```

Comment:
Instead of saying what he *doesn't* want to do, say what he *does*, and contrast that with what he doesn't.
```suggestion
			`	"Hoping to join the research teams on Ring of Wisdom, I am," he says as you two walk to your ship, explaining the reason he needs to study so much is because he is aiming to work in the scientific branch of the government, instead of the more military-focused side.`
```